src/tools/_tools.font-size.scss

```diff
@@ -2,6 +2,57 @@
 //     #FONT-SIZE
 // \*------------------------------------*/
 
+/// Generates a font-size declaration and (optionally) an equivalent
+/// line-height.
+///
+/// @param {number | length } $ms [0] - Determine the font-size by using a value
+///   of the modular scale.
+/// @param {number | auto | "normal" | "inherit" | "none" | "false"} $line-height
```

Comment:
`{number | string}`

Reply:
`string` as a param wouldn’t be correct, as that would create the impression that every string could be used, where in fact only these specific keywords are allowed. Any other value would provoke a Sass error.

Reply:
They are still strings. You can add the allowed keywords in the description

Reply:
Copy that.